src/hb-ot-var-fvar-table.hh

```diff
@@ -280,6 +280,32 @@ struct fvar
     return axisCount;
   }
 
+  void collect_name_ids (hb_set_t *nameids) const
+  {
+    if (!has_data ()) return;
+
+    + get_axes ()
+    | hb_apply ([&] (const AxisRecord& _) { hb_set_add (nameids, (unsigned) _.axisNameID); })
+    ;
+
+    + hb_range ((unsigned) instanceCount)
+    | hb_apply ([&] (const unsigned _) 
```

Comment:
change apply to be hb_map, and keep the lambda then add an hb_sink after to add to the set, likewise below.